lib/src/mistralai_client_dart_base.dart

```diff
@@ -1,7 +1,12 @@
+import 'dart:convert';
 import 'package:http/http.dart' as http;
 import 'package:http/retry.dart' as retry;
 import 'package:mistralai_client_dart/src/models/models.dart';
 
+class _MistralAPIEndpoints {
+  static const String listModels = '/v1/models';
+}
```

Comment:
I'm not sure if we need it as a separate const. I don't think there will be more than one usage atm. My suggestion would be to keep it locally in `listModels` method. @KacperKluka wdyt?

Reply:
I think we should organize things nicely and put it in a dedicated place, so it's easier to maintain later 😉 Check my comment here https://github.com/nomtek/mistralai_client_dart/pull/12#discussion_r1450361537

Reply:
I'm move the conversion to https://github.com/nomtek/mistralai_client_dart/pull/12#discussion_r1450361537 + close this one.